lib/internal/url.js

```diff
@@ -312,6 +312,7 @@ class URL {
     // toUSVString is not needed.
     input = `${input}`;
     if (base !== undefined &&
+        base !== null &&
```

Comment:
According to the current specification of URL, it might not be needed to checking base as like instanceof since base is guaranteed to be a USVString. Also, in this case, it should display an error for parsing `base = null`, so I think we can simply write like the following:
```js
if (base !== undefined) {
  base = new URL(base);
}
```

Refs: https://github.com/jsdom/whatwg-url/blob/master/lib/URL-impl.js#L12-L17
Spec: https://github.com/whatwg/url/blob/master/url.bs#L2592-L2599

Reply:
Yeah you're right, the undefined check is also incorrect

Reply:
Thanks for your quick update!